src/Controller/AboutController.php

```diff
@@ -270,17 +271,22 @@ public function peopleAction(Request $request, string $type) : Response
                 $editorInChief = $leadership->filter(function (Person $person) {
                     return 'Editor-in-Chief' === $person->getTypeLabel();
                 });
-                $foundingEditorInChief = $leadership->filter(function (Person $person) {
-                    return 'Founding Editor-in-Chief' === $person->getTypeLabel();
-                });
+                // Load Founding Editor-in-Chief.
+                $foundingEditorInChief = $people->get('6d42f4fe')
+                    ->then(function (Person $person) {
+                        return new ArraySequence([$person]);
+                    })
+                    ->otherwise(function () {
+                        return new EmptySequence();
```

Comment:
I think this could be stricted, some other code in this controller does:
```
    ->otherwise($this->softFailure('Failed to load ...', new EmptySequence()));
```
which I think adds a `WARN` log rather than just ignoring the failure.